src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -207,6 +207,12 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 chatBox.clear();
             }
 
+            // Force refresh of the following UI components which are time sensitive
+            // Overdue reminders should be displayed differently
+            this.reminderListPanel.refresh();
+            // Past meetings should be filtered out
```

Comment:
Is this something that was implemented in this PR? I didn't experience any filtering out of past meetings when executing commands to refresh the display.

Reply:
Not yet! Just adding this in here so that I won't forget. Sorry for the confusion

Reply:
Great, thanks for clarifying! Will approve and merge.